app/views/posts/index.html.erb

```diff
@@ -1 +1,7 @@
-<h1>Index posts</h1>
+<section>
+  <%= render partial:'partials/user', locals: {user: @user} %>
+  <div class="posts">
```

Comment:
Please create a method and use it to render 5 most recent comments in the `app/views/posts/index.html.erb`